src/main/kotlin/study/view/InputView.kt

```diff
@@ -0,0 +1,17 @@
+package study.view
+
+class InputView {
+    companion object {
+        fun askCarCount(): Int {
+            println("자동차 대수는 몇 대인가요?")
+            return runCatching { readln().toInt() }.getOrElse { throw IllegalArgumentException(INPUT_ERROR_MESSAGE) }
```

Comment:
```suggestion
            return readln().toIntOrNull() ?: throw IllegalArgumentException(INPUT_ERROR_MESSAGE)
```
runCatching 을 사용하는것도 좋지만 간단하게는 위처럼 표현할수도 있을것 같아요 😄